tests/ui/consts/const_waker.rs

```diff
@@ -0,0 +1,28 @@
+//@ check-pass
```

Comment:
What exactly is this test verifying? Usually `tests/ui` is for testing compiler behavior (or tests that require specific execution). Tests for library behavior is in `library/core/tests`.

Reply:
I think it's for verifying the const-stability. I have moved it to `library/core/tests`.